webpack/components/extensions/HostDetails/DetailsTabCards/ImageModeCard.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+import {
+  DescriptionList,
+  DescriptionListGroup,
+  DescriptionListDescription as Dd,
+  DescriptionListTerm as Dt,
+} from '@patternfly/react-core';
+import CardTemplate from 'foremanReact/components/HostDetails/Templates/CardItem/CardTemplate';
+import FontAwesomeImageModeIcon from '../../../../components/extensions/Hosts/FontAwesomeImageModeIcon';
+
+const ImageModeCard = ({ isExpandedGlobal, hostDetails }) => {
+  const imageMode = hostDetails?.content_facet_attributes?.bootc_booted_image;
+  const header = (
+    <>
+      <span style={{ marginRight: '0.5rem' }}>{__('Image mode host')}</span>
+      <FontAwesomeImageModeIcon />
+    </>
+  );
+  if (!imageMode) return null;
+  return (
+    <CardTemplate
+      header={header}
+      expandable
+      masonryLayout
+      isExpandedGlobal={isExpandedGlobal}
+    >
+      <DescriptionList isHorizontal>
+        <DescriptionListGroup>
+          <Dt>{__('Running image')}</Dt>
+          <Dd>{hostDetails.content_facet_attributes.bootc_booted_image}</Dd>
+          <Dt>{__('Running image digest')}</Dt>
+          <Dd>{hostDetails.content_facet_attributes.bootc_booted_digest}</Dd>
+
+          <Dt>{__('Staged image')}</Dt>
```

Comment:
For any field that is empty, should we display a dash `-` like in the mock image?

![image](https://github.com/user-attachments/assets/7b9bac59-0950-425f-b8e0-b1ced8dcc558)

Reply:
This seems to work well:

```
diff --git a/webpack/components/extensions/HostDetails/DetailsTabCards/ImageModeCard.js b/webpack/components/extensions/HostDetails/DetailsTabCards/ImageModeCard.js
index 7ec258bf77..ce281d6274 100644
--- a/webpack/components/extensions/HostDetails/DetailsTabCards/ImageModeCard.js
+++ b/webpack/components/extensions/HostDetails/DetailsTabCards/ImageModeCard.js
@@ -19,6 +19,8 @@ const ImageModeCard = ({ isExpandedGlobal, hostDetails }) => {
     </>
   );
   if (!imageMode) return null;
+  const getValueOrDash = value => (value || '-');
+
   return (
     <CardTemplate
       header={header}
@@ -29,24 +31,24 @@ const ImageModeCard = ({ isExpandedGlobal, hostDetails }) => {
       <DescriptionList isHorizontal>
         <DescriptionListGroup>
           <Dt>{__('Running image')}</Dt>
-          <Dd>{hostDetails.content_facet_attributes.bootc_booted_image}</Dd>
+          <Dd>{getValueOrDash(hostDetails.content_facet_attributes.bootc_booted_image)}</Dd>
           <Dt>{__('Running image digest')}</Dt>
-          <Dd>{hostDetails.content_facet_attributes.bootc_booted_digest}</Dd>
+          <Dd>{getValueOrDash(hostDetails.content_facet_attributes.bootc_booted_digest)}</Dd>
 
           <Dt>{__('Staged image')}</Dt>
-          <Dd>{hostDetails.content_facet_attributes.bootc_staged_image}</Dd>
+          <Dd>{getValueOrDash(hostDetails.content_facet_attributes.bootc_staged_image)}</Dd>
           <Dt>{__('Staged image digest')}</Dt>
-          <Dd>{hostDetails.content_facet_attributes.bootc_staged_digest}</Dd>
+          <Dd>{getValueOrDash(hostDetails.content_facet_attributes.bootc_staged_digest)}</Dd>
 
           <Dt>{__('Available image')}</Dt>
-          <Dd>{hostDetails.content_facet_attributes.bootc_available_image}</Dd>
+          <Dd>{getValueOrDash(hostDetails.content_facet_attributes.bootc_available_image)}</Dd>
           <Dt>{__('Available image digest')}</Dt>
-          <Dd>{hostDetails.content_facet_attributes.bootc_available_digest}</Dd>
+          <Dd>{getValueOrDash(hostDetails.content_facet_attributes.bootc_available_digest)}</Dd>
 
           <Dt>{__('Rollback image')}</Dt>
-          <Dd>{hostDetails.content_facet_attributes.bootc_rollback_image}</Dd>
+          <Dd>{getValueOrDash(hostDetails.content_facet_attributes.bootc_rollback_image)}</Dd>
           <Dt>{__('Rollback image digest')}</Dt>
-          <Dd>{hostDetails.content_facet_attributes.bootc_rollback_digest}</Dd>
+          <Dd>{getValueOrDash(hostDetails.content_facet_attributes.bootc_rollback_digest)}</Dd>
         </DescriptionListGroup>
       </DescriptionList>
     </CardTemplate>
```